spec-0009/index.md

```diff
@@ -0,0 +1,185 @@
+for participation helps maintain a continuous stream of contributions.
+
+This SPEC outlines the definition of governance, various open-source governance
+models, the steps to choose the right governance model, and what needs to be
+done post-adoption.
+
+Please note that these are recommendations and not bylaws. If you have
+suggestions/thoughts, we're more than happy to consider them.
+
+### Core Project Endorsement
+
+<!--
+Briefly discuss what it means for a core project to endorse this SPEC.
+-->
+
+### Ecosystem Adoption
+
+<!--
+Briefly discuss what it means for a project to adopt this SPEC.
+-->
+
+## Implementation
+
+### Choosing a governance model
+
+When you define governance for a project, you need to identify a few things:
+
+- What roles does maintainers, core team, contributors, and community members
+  play in the project?
+- What duties, privileges, and authority are associate with each role?
+- How do people get assigned to and removed from roles?
+- How decisions are made and documented within the project?
+- What are the methods for resolving conflicts?
+
+There are few **pre-defined models** for governance used across open-source
+projects. We'll mention them here for reference:
+
+- BDFL
+  - BDFL stands for 'Benevolent Dictator for Life.' Under this structure, one
+    person (usually the project's initial author) has the final say on all
+    major project decisions. Smaller projects are BDFL by default because
+    there are only one or two maintainers.
+  - BDFL model template → http://oss-watch.ac.uk/resources/benevolentdictatorgovernancemodel
+- Meritocracy / Do-ocracy
+  - Decisions are made by contributors who have demonstrated merit through
+    their contributions to the project. Merit is often gained through
+    contributions of code, documentation, or other significant efforts.
+  - Meritocracy model template → http://oss-watch.ac.uk/resources/meritocraticgovernancemodel
+- Self appointing council / committee / board
+  - This model appoints a board or committee to govern various aspects of a
+    project. Those groups are often called the steering committee, committer
+    council, technical operation committee, board of directors, etc. This
+    model might be helpful for projects that need a sponsoring foundation,
+    and establishing electoral mechanisms is challenging.
+- Electoral
+  - The electoral governance model in open-source projects is a structured
+    approach in which the community elects the leadership or decision-making
+    body. This model aims to ensure that those who guide the project's
+    direction and make crucial decisions are chosen democratically,
+    representing the broader community's interests.
+- Foundation backed
+  - A non-profit foundation oversees the project, and its governance
+    structures often include a board of directors, technical committees, and
+    working groups.
```

Comment:
I've removed the 'Corporate backed / Single Vendor governance model' in https://github.com/scientific-python/specs/pull/323/commits/750a088acb40e843817d94f770259618ae6f8222 as discussed with @jarrodmillman over the summit.

Should we also remove the 'Foundation-backed' model? I went through [Jupyter's old governance model](https://jupyter.org/governance/archive/governance.html) and realised there used to be a subcommittee for non-profit (NumFOCUS) to manage the funding.

I understand that it's the old model and not the current one, but I would like to know if it's helpful to keep it here. Are there any scientific Python projects out there that subscribe to a similar model?